package.json

```diff
@@ -141,6 +141,7 @@
     "blueimp-load-image": "~2.12.2",
     "classnames": "~2.2.5",
     "enumerate-devices": "^1.1.0",
+    "eslint-plugin-jest": "^23.6.0",
```

Comment:
I think this should go to devDependencies.

Reply:
Oh, I thought I had run it with `--save-dev`. Good spot!